dns_sd.c

```diff
@@ -98,6 +98,14 @@ static int dnssd_fill_context_info(struct iio_context_info *info,
 	}
 
 	ctx = network_create_context(uri);
+
+	/* Sometimes, on Windows local/private networks, numeric IP address
+	 * fails but hostname works (don't ask me why), so try both.
+	 */
```

Comment:
Wouldn't it be better to try the hostname first then? Or does that cause problems on other places?

Reply:
yeah, this causes problems in cases where the host name can't be looked up. On pre-Windows 10  - for the OS to do lookups for *.local addresses, you need to have bonjour installed - otherwise ip addresses are the only way it works.

Reply:
this seems to only be a problem on private/local networks (unconfigured cross over cables) when you plug it in, and then check within the first 30 seconds or so, (at least that is when I noticed it); so I think IP addresses failing is the minor case...

Reply:
Shouldn't that be protected with a `#ifdef _WIN32` guard then?